utopian-io/v2.utopian.io
# Edit a user profile basic information

### Verify that


Please provide a test plan so that the reviewers can see how you tested your work and/or the different acceptance criteria described in the task.

### **Test plan**

_Update your profile_

* Change your information
* Save
* Reload

**Expected result:** You should see the updated values

_BONUS: Update the avatar_

* Change the avatar
* Save

**Expected result:** the avatar in the toolbar is also updated
